.env.development

```diff
@@ -6,3 +6,6 @@ REACT_APP_GITHUB_API_URL=http://localhost:8002
 
 # This is for the slack API
 REACT_APP_SLACK_API_URL=http://localhost:8000
+
```

Comment:
@michaelmurray6298 I'd like to see the URL be: http://localhost:8003. Please update it here and in sheets-spa